app/classes/document_query.rb

```diff
@@ -133,6 +139,13 @@ def suggestion_highlight
     }
   end
 
+  #Temporary fix for https://github.com/elastic/elasticsearch/issues/34282
+  def query_without_stopwords
+    query_without_stopwords = query
+    DEFAULT_STOPWORDS.each{|word| query_without_stopwords.remove!(/\b#{word}\b/)}
+    query_without_stopwords.squish
+  end
```

Comment:
Why not just do this instead of looping through? 
query.split(/ +/) - DEFAULT_STOPWORDS
or if query is already an array
query - DEFAULT_STOPWORDS 
It will squash this method down to 1 line.

Reply:
Actually it be something like this (query.split(/ +/) - DEFAULT_STOPWORDS).join(" ")  since you have to revert it back to a string :)

Reply:
Yeah, I did that. I am also downcasing the query so that "THIS IS A TEST" will be stripped to "test". 

It occurred to me that punctuation could also be a gotcha, as "this test there." would not have "there" stripped out. However, I reviewed the list of 200k queries that I pulled, and punctuation is few and far between. I think we should err on the side of simplicity, as stripping out punctuation could result in queries like "form 20.4" becoming "form 204".